ngraph_bridge/ie_backend_engine.h

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
```

Comment:
Minor nit in terms of naming: I'd prefer to call this `Backend`. The "IE" prefix is unnecessary since that's the only type of backends we expect to interface with. The files can similarly be renamed to be:

```
backend.{h,cc}
backends/hddl.{h,cc}
backends/myriad.{h,cc}
```